React/Base/RCTUtils.m

```diff
@@ -385,6 +399,12 @@ CGFloat RCTFloorPixelValue(CGFloat value)
   return floor(value * scale) / scale;
 }
 #else // [macOS
+CGSize RCTViewportSize()
+{
+  NSScreen* screen = [NSScreen mainScreen];
```

Comment:
Can we add an issue to revisit using the mainScreen always for Fabric and try to do better for mult-window? May not be actionable for a while, but I'd like it documented.

Reply:
Yes, I'll add an issue for this. Let's discuss the multi-window solution since we'd love to get feedback try and upstream our implementation.